zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/ZkTestBase.java

```diff
@@ -0,0 +1,148 @@
+package org.apache.helix.zookeeper.impl;
```

Comment:
There is ZkTestBase in Helix common before. So what is the strategy here. 
1/ Are we going to remove that ZkTestBase from Helix common and associated test?
2/ Move ZkTestBase in Helix common and associated test here?
3/ Have two ZKTestBase s and their related test in parallel?

Reply:
1. No, this is a ZkTestBase for zookeeper-api module. The name of the class is the same, but it does different things.
2. No.
3. Yes. zookeeper-api needs to have tests and this ZkTestBase contains the ZK environment setup logic.